src/main/java/org/fcrepo/spec/testsuite/authz/WebACLinking.java

```diff
@@ -63,26 +79,29 @@ public void linkToAclOnCreation(final String uri) {
     @Parameters({"param1"})
     public void conflictIfNotSupportingAclLink(final String uri) {
         final TestInfo info = setupTest("5.4-B", "conflictIfNotSupportingAclLink",
-                "The server must reject the request and respond with a 4xx or 5xx range status code, such as " +
-                        "409 (Conflict) if it isn't able to create the LDPR with the specified LDP-RS as the ACL.",
-                "https://fedora.info/2018/06/25/spec/#link-acl-on-create", ps);
-    }
+                                        "The server must reject the request and respond with a 4xx or 5xx range " +
+                                        "status code, such as " +
+                                        "409 (Conflict) if it isn't able to create the LDPR with the specified LDP-RS" +
+                                        " as the ACL. " +
+                                        "In that response, the restrictions causing the request to fail must be " +
+                                        "described in a " +
+                                        "resource indicated by a rel=\"http://www.w3.org/ns/ldp#constrainedBy\" link " +
+                                        "in the Link " +
+                                        "response header",
+                                        "https://fedora.info/2018/06/25/spec/#link-acl-on-create", ps);
+
+        //create a resource to be used for the acl link
+        final Response aclResponse = createBasicContainer(uri, info.getId() + "-acl");
+        final String aclUri = getLocation(aclResponse);
+
+        //PUT the new Resource
+        final String aclLinkValue = "<" + aclUri + ">; rel=\"acl\"";
+        final Response resource = doPutUnverified(uri, Headers.headers(new Header("Link", aclLinkValue)), "test body");
+
+        if (resource.getStatusCode() >= 400) {
+            confirmPresenceOfConstrainedByLink(resource);
+        }
```

Comment:
There should probably be an `else` block the verifies response code in the `successRange`.